### DISCUSSION TITLE
posts/robtop-previewed-geometry-dash-2-2-keyframe-system/

### DISCUSSION THREAD

**Discussion by giscus** [MoldyMacaroniX/gd-website#64] [Comments]
# posts/robtop-previewed-geometry-dash-2-2-keyframe-system/

RobTop released a new Geometry Dash 2.2 preview, showcasing the new 2.2 keyframe system.

https://www.dashword.net/posts/robtop-previewed-geometry-dash-2-2-keyframe-system/

**Comment by MoldyMacaroniX** [comment]
What do you think of this new system? Do you think it is too overpowered? Let me know!
